javascript/ql/src/semmle/javascript/DefUse.qll

```diff
@@ -182,14 +182,26 @@ class VarDef extends ControlFlowNode {
   /** Gets a variable defined by this node, if any. */
   Variable getAVariable() { result = getTarget().(BindingPattern).getAVariable() }
 
+  private AST::ValueNode getTargetSource(Expr target) { defn(this, target, result) }
+
   /**
    * Gets the source of this definition, that is, the data flow node representing
    * the value that this definition assigns to its target.
    *
    * This predicate is not defined for `VarDef`s where the source is implicit,
    * such as `for-in` loops or parameters.
```

Comment:
Could you add a paragraph explaining that it's also not defined for destructuring definitions?